### ISSUE TITLE
Forgot to push VnV Report?

### ISSUE THREAD

**Issue by smiths** [Tan630/Agolearn#57]
@Tan630 did you forget to push the VnV report?  The Reflection document is also missing.

**Comment by Tan630** [comment]
Sorry about that - I have added the [VnV report](https://github.com/Tan630/Agolearn/blob/main/docs/VnVReport/VnVReport.pdf) (in 6142311cbc77bb38302ee3b4ce56ab34a929dd71) and presentations (in 2a3e0fd195bbe5ed72aac9b56420986addff6920). I have changed the version to reflect the upload time.

I did forget to write the reflection - I deleted the folder earlier, thinking that it is part of the capstone components. I _thought_ that [Reflection.md](https://github.com/Tan630/Agolearn/blob/main/Reflection.md) could be my reflection, not aware that there is a template for that. I will write the reflection, then add it as a late submission.
